debian/manpages/snapcraft.1

```diff
@@ -0,0 +1,87 @@
+.TH snapcraft 1 "19 July 2016" "version 2.13"
+.SH NAME
+Snapcraft - Snapcraft is a delightful packaging tool
+.SH SYNOPSIS
+snapcraft [options] <command>
+
+snapcraft (-h | --help)
+
+snapcraft --version
+.SH DESCRIPTION
+Snapcraft helps you assemble a whole project in a single tree out of many pieces. It can drive a very wide range of build and packaging systems, so that you can simply list all the upstream projects you want and have them built and installed together as a single tree.
```

Comment:
It's generally good style in troff to start a new line after each piece of sentence-final punctuation.  The toolchain will use slightly better typography in some output formats if you do that.